SPECS/kernel/kernel.spec

```diff
@@ -224,9 +232,9 @@ rm -rf %{buildroot}/lib/modules/%{uname_r}/source
 rm -rf %{buildroot}/lib/modules/%{uname_r}/build
 
 find . -name Makefile* -o -name Kconfig* -o -name *.pl | xargs  sh -c 'cp --parents "$@" %{buildroot}/usr/src/linux-headers-%{uname_r}' copy
-find arch/${archdir}/include include scripts -type f | xargs  sh -c 'cp --parents "$@" %{buildroot}/usr/src/linux-headers-%{uname_r}' copy
```

Comment:
What was happening before this change? Did we always just find nothing? I'm curious what's the impact of this change is going to be.

Reply:
First and third lines were failing because the `arch//include` directory does not exist. The second line was running a broader search under the entire `arch` directory, instead of just the specific architecture. Here is a build log:

```
time="2020-10-05T10:19:06Z" level=debug msg="+ find arch//include include scripts -type f"
time="2020-10-05T10:19:06Z" level=debug msg="+ xargs sh -c 'cp --parents \"$@\" /usr/src/mariner/BUILDROOT/kernel-5.4.51-9.cm1.x86_64/usr/src/linux-headers-5.4.51-9.cm1' copy"
time="2020-10-05T10:19:06Z" level=debug msg="find: 'arch//include': No such file or directory"
time="2020-10-05T10:19:07Z" level=debug msg="+ xargs sh -c 'cp --parents \"$@\" /usr/src/mariner/BUILDROOT/kernel-5.4.51-9.cm1.x86_64/usr/src/linux-headers-5.4.51-9.cm1' copy"
time="2020-10-05T10:19:07Z" level=debug msg="++ find arch/ -name include -o -name scripts -type d"
time="2020-10-05T10:19:07Z" level=debug msg="+ find arch/um/scripts arch/um/include arch/microblaze/include arch/alpha/include arch/hexagon/include arch/nds32/include arch/xtensa/platforms/xt2000/include arch/xtensa/platforms/iss/include arch/xtensa/platforms/xtfpga/include arch/xtensa/variants/test_kc705_hifi/include arch/xtensa/variants/csp/include arch/xtensa/variants/dc233c/include arch/xtensa/variants/de212/include arch/xtensa/variants/dc232b/include arch/xtensa/variants/fsf/include arch/xtensa/variants/test_kc705_be/include arch/xtensa/variants/test_mmuhifi_c3/include arch/xtensa/include arch/parisc/include arch/arm64/include arch/h8300/include arch/nios2/include arch/openrisc/include arch/riscv/include arch/c6x/include arch/sh/include arch/mips/include arch/csky/include arch/arc/plat-eznps/include arch/arc/include arch/s390/scripts arch/s390/include arch/arm/mach-exynos/include arch/arm/mach-ebsa110/include arch/arm/mach-omap1/include arch/arm/mach-davinci/include arch/arm/plat-pxa/include arch/arm/plat-omap/include arch/arm/mach-spear/include arch/arm/plat-orion/include arch/arm/mach-s3c64xx/include arch/arm/plat-versatile/include arch/arm/mach-dove/include arch/arm/mach-s3c24xx/include arch/arm/mach-omap2/include arch/arm/mach-footbridge/include arch/arm/mach-ixp4xx/include arch/arm/mach-ep93xx/include arch/arm/mach-pxa/include arch/arm/plat-samsung/include arch/arm/mach-rpc/include arch/arm/include arch/arm/mach-sa1100/include arch/arm/mach-iop32x/include arch/sparc/include arch/m68k/include arch/ia64/scripts arch/ia64/include arch/powerpc/include arch/unicore32/include arch/x86/include -type f"
time="2020-10-05T10:19:07Z" level=debug msg="+ find arch//include Module.symvers include scripts -type f"
time="2020-10-05T10:19:07Z" level=debug msg="+ xargs sh -c 'cp --parents \"$@\" /usr/src/mariner/BUILDROOT/kernel-5.4.51-9.cm1.x86_64/usr/src/linux-headers-5.4.51-9.cm1' copy"
time="2020-10-05T10:19:07Z" level=debug msg="find: 'arch//include': No such file or directory"
```

Looking at the `diff` of the two `kernel-devel` packages before and after this change, the only difference seems to be that the new version does not include the headers under `arch/<other-arch>` directories. I am making the PR into `dev` instead of `1.0-dev` for now since the package layout changes slightly.